### ISSUE TITLE
Not working for inline Latex

### ISSUE THREAD

**Issue by filiptrplan** [DearVikki/obsidian-cloze-plugin#20]
When I create a Cloze with inline Latex, the Latex is always visible, even if the other clozes are toggled off and I can't toggle it with my mouse.
Minimum reproducible example:
```
<span class="cloze-span">$test$</span>
```
The weird thing is that if I place two chacters on either side it works:
```
<span class="cloze-span">ab $132$ ab</span>
```
But only one doesn't:
```
<span class="cloze-span">a $132$ a</span>
```

**Comment by DearVikki** [comment]
Hey @filiptrplan, thanks for letting me know about this 🐞 . Yeah, I think I already figured out the reason behind it..looks like I need to brush up on my regex skills, haha. I'll make sure to fix it asap in the next version!

**Comment by DearVikki** [comment]
Hi @filiptrplan, already fixed in the latest release (version 0.1.13) and I hope it works all well for you!
